src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Components/SortableHeaderCell/Default.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@using Geta.NotFoundHandler.Admin.Areas.GetaNotFoundHandlerAdmin.Pages.Models
+@model Geta.NotFoundHandler.Admin.Areas.GetaNotFoundHandlerAdmin.Pages.Components.SortableHeaderCell.SortableHeaderCellViewModel
+
+<a asp-route-sortColumn="@Model.Key"
+   asp-route-sortDirection="@(Model.SortDirection == SortDirection.Ascending ? SortDirection.Descending : SortDirection.Ascending)"
+   style="white-space: nowrap">
+    @Model.DisplayName
+
+    <span data-feather="@(Model.IsActive ? Model.SortDirection == SortDirection.Ascending ? "arrow-up" : "arrow-down" : "arrow-up")">
```

Comment:
The glyph will be displayed even for columns that have not been specifically sorted.